README.md

```diff
@@ -127,62 +127,87 @@ APP_NIGHTFALL_MNEMONIC=
 APP_TOKEN_ADDRESS_1=0x4f3c4F8D4575Cf73c2FAf9F36cc505e19E65B9C0
 ```
 
+```
+# Contents of .env.goerli
+APP_BLOCKCHAIN_WEBSOCKET_URL=your web3 socket provider
+APP_CLIENT_API_URL=http://localhost:8080
+APP_ETH_PRIVATE_KEY=your Ethereum private key
+APP_NIGHTFALL_MNEMONIC=your Nightfall mnemonic
+# Goerli MATIC
+APP_TOKEN_ADDRESS_1=0x499d11E0b6eAC7c0593d8Fb292DCBbF815Fb29Ae
+```
```

Comment:
Because you are keeping some comments for the Goerli vars, I was wondering if we could update the Ganache ones to indicate that some of the given details relate nightfall_3 local deployment:

```
# Contents of .env.ganache
APP_BLOCKCHAIN_WEBSOCKET_URL=ws://localhost:8546
APP_CLIENT_API_URL=http://localhost:8080

# Nightfall_3 User 1
APP_ETH_PRIVATE_KEY=0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e

APP_NIGHTFALL_MNEMONIC=

# Nightfall_3 ERC20Mock contract address in ganache
# Make sure to monitor the local deployment and double-check the contract address
APP_TOKEN_ADDRESS_1=0xa8473bEF03cBE50229a39718CBDC1fdee2F26b1a
```